subjects/Elf/ARM/switch/switch.reko/switch_text.c

```diff
@@ -7,6 +7,7 @@
 // 00008334: void _start(Stack int32 dwArg00, Stack (ptr32 Eq_n) ptrArg08, Stack (ptr32 Eq_n) ptrArg0C, Stack (ptr32 void) ptrArg10)
 void _start(int32 dwArg00, void (* ptrArg08)(), void (* ptrArg0C)(), void * ptrArg10)
 {
+	ptr32 fp;
```

Comment:
This `fp` is there because of the `fp + 4` on the next line, right? Hopefully that will go away when `fp + 4` is replaced with `&tArg04`

Reply:
Yes it should go away when `fp + 4` is replaced with `&tArg04`